### ISSUE TITLE
bug(mat-expansion-panel): In high contrast mode, keyboard focus not visible around panel

### ISSUE THREAD

**Issue by dwaynelavon** [angular/components#21544]
#### Reproduction

Stackblitz: https://stackblitz.com/angular/drrqqddrvej?file=src%2Fapp%2Fexpansion-overview-example.ts
 
#### Expected Behavior

What behavior were you expecting to see?
1. Enable Windows high contrast mode in settings
2. Open Firefox
4. Create a mat-expansion-panel (Stackblitz above)
4. Use keyboard to tab to the expansion panel and observe focus indicator

#### Actual Behavior

What behavior did you actually see?
Keyboard focus indicator is not visible around expansion panel

#### Environment

- Angular: 11.0.3
- CDK/Material: 11.0.3
- Browser(s): Firefox
- Operating System (e.g. Windows, macOS, Ubuntu):  Windows
